valkyrie.drush.inc

```diff
@@ -121,7 +121,11 @@ function valkyrie_drush_command() {
     'aliases' => array('vup'),
     'bootstrap' => DRUSH_BOOTSTRAP_DRUSH, // No bootstrap at all.
   );
-
+  $items['valkyrie-upgrade'] = array(
+    'description' => 'Upgrade global Valkyrie extension and all of the submodules.',
+    'aliases' => array('vug'),
```

Comment:
This one is a little ambiguous to me, needing some clearing up on it.

If we need to download the latest release tarball, I feel like we will need something like `drush pm-releases valkyrie` or an equivalent hook to give us the title of the latest release and pop it at the end of http://ftp.drupal.org/files/projects/valkyrie- and then add .tar.gz after that.
Right now `drush pm-releases valkyrie` doesn't show any releases for Valkyrie as of yet, and It seems like we are reinventing `drush dl` with this as well.

Possibly I overcomplicated this, and we are going to explicitly call `drush dl valkyrie` when `drush valkyrie-site-upgrade` is ran, and just overwrite the old project, but that seems redundant to me here.